README.md

```diff
@@ -33,6 +33,7 @@ If you are developing a project that uses new CSS language features and must wor
 - You can use any features from [cssnext](http://cssnext.io/)
 - Enables [postcss-import](https://github.com/postcss/postcss-import) so that `@import` statements are inlined
 - Enables [postcss-mixins](https://github.com/postcss/postcss-mixins) so that you can define mixins
+- Enables [postcss-conditionals](https://github.com/andyjansson/postcss-conditionals) so that you can define @if statements
```

Comment:
Could you put @if under backticks? `@if`

Reply:
done :)